requirements.txt

```diff
@@ -41,6 +41,7 @@ terminaltables==3.1.0
 sanic==19.9.0
 sanic-cors==0.9.9.post1
 sanic-jwt==1.3.2
+sanic-plugins-framework==0.8.2
```

Comment:
why?

Reply:
Lot's of trivial tests fail if we don't do that. See https://rasa-hq.slack.com/archives/C36SS4N8M/p1578908846011300

Reply:
should be already fixed on master by tanja, so you can remove the duplicate line

Reply:
Ah. Didn't see that it's merged already. I now merged with master and resolved the conflict.